### PR TITLE
Update summary report with header

### PR DESCRIPTION
**Pull request by rtesra**

### CODE REVIEW

File: inst/report/summary_report.Rmd [hunk 1]
```diff
@@ -679,3 +670,6 @@ x4 %>% gt::gt(rowname_col = "area_name") %>%
 
 ```
 
+### **Naomi methods**
+
+[place holder text for Naomi methods]
```
Comment (rtesra) [inline inst/report/summary_report.Rmd:675]: @jeffeaton You can add the methods in here as raw text

File: R/outputs.R [hunk 2]
```diff
@@ -672,18 +672,28 @@ save_output <- function(filename, dir,
...
-  report_path_dir <- normalizePath(dirname(report_path), mustWork = TRUE)
   report_filename <- basename(report_path)
+  report_path_dir <- normalizePath(dirname(report_path), mustWork = TRUE)
   output_zip_path <- normalizePath(output_zip, mustWork = TRUE)
-  rmarkdown::render(rmd_path, params = list(
+  ## Render uses relative paths to locate the html file and pacakge author
+  ## advises against using output_dir see:
+  ## https://github.com/rstudio/rmarkdown/issues/587#issuecomment-168437646
+  ## so set up a temp directory with all report sources and generate from there
+  tmpd <- tempfile()
```
Comment (richfitz) [inline R/outputs.R:683]: You might use withr here for these , but this is ok too

File: R/outputs.R [hunk 3]
```diff
@@ -672,18 +672,28 @@ save_output <- function(filename, dir,
...
+  file.copy(list.files(system_file("report"), full.names = TRUE), ".")
+
+  rmarkdown::render("summary_report.Rmd", params = list(
     output_zip = output_zip_path),
-    output_dir = report_path_dir,
     output_file = report_filename,
     quiet = quiet
   )
 
+  file.copy(report_filename, report_path_dir)
```
Comment (richfitz) [inline R/outputs.R:697]: I advise using stopifnot() around file.copy as it does not error on failure. Or use a function from `fs`, which generally has better behaviour.

Do we not need the whole directory?

**Review by richfitz — CHANGES_REQUESTED** [review]
Couple of small comments here

File: R/outputs.R (hunk 2 above)
Comment (r-ash) [reply R/outputs.R:683]: `withr` is nicer, I have switched that over

File: R/outputs.R (hunk 3 above)
Comment (r-ash) [reply R/outputs.R]: Thanks, switched to use `fs` we only need to copy the single file. The rendering pulls the images & banner html into the output document so it is all available in the single file.

**Review by richfitz — APPROVED** [review]

**Pull request merged**